src/index.js

```diff
@@ -1,6 +1,8 @@
 if(!global._babelPolyfill) { require('babel-polyfill'); }
 
-import _ from 'lodash';
+import { isEmpty, pick, extend, omit } from 'lodash';
```

Comment:
I don't think this is how it works unless Babel added Tree shaking. We'll probably have to do

```
import isEmpty from 'lodash/isempty';
import extend from 'lodash/extend';
```

etc. Or to make it much easier and not having to change the rest of the code:

```js
const _ = {
  extend: require('lodash/extend'),
  isEmpty: require('lodash/is-empty')
}
```